code/controllers/subsystem/job.dm

```diff
@@ -24,7 +24,8 @@ SUBSYSTEM_DEF(job)
 		"Research Director" = 3,
 		"Chief Engineer" = 4,
 		"Chief Medical Officer" = 5,
-		"Head of Security" = 6)
+		"Head of Security" = 6,
+		"Quartermaster" = 7)
```

Comment:
Hos should always be the last in the list.
Plus this requires wiki changes.

Reply:
I have changed that with https://github.com/BeeStation/BeeStation-Hornet/pull/6833/commits/60363fa432fd71ffea7e1b6a718122204c1a4711, for wiki changes I've made a sandbox page for what to change.